src/platform/android/AndroidChipPlatform-JNI.cpp

```diff
@@ -60,6 +60,9 @@ static bool JavaBytesToUUID(JNIEnv * env, jbyteArray value, chip::Ble::ChipBleUU
 namespace {
 JavaVM * sJVM = nullptr;
 JniGlobalReference sAndroidChipPlatformExceptionCls;
+jclass sCallbackClass;
```

Comment:
for global refer, please use JniGlobalReference instead of jclass

Reply:
Thank you for your advice. I will change the `sCallbackClass` to `callbackClass` local variable and change the type of `sOnLogMessageMethod` from `jobject` to `JniGlobalReference`.